apps/arkmarket/src/components/cells/activity-price-cell.tsx

```diff
@@ -0,0 +1,19 @@
+import { PriceTag } from "@ark-market/ui/price-tag";
+import type { CollectionActivity, PortfolioActivity, PortfolioOffers } from "~/types";
+import { TableCell } from "@ark-market/ui/table";
+
+interface PriceCellProps {
+  activity: CollectionActivity | PortfolioActivity | PortfolioOffers;
+}
+
+export default function PriceCell({ activity }: PriceCellProps) {
+  return (
+    <TableCell className="flex w-[25%]">
```

Comment:
remove `w-[25%]`, it breaks this page for example: `wallet/0x64ae291c8482c273828957239017c3dda0ec12ef27f76c9c9b043740cdd9137?activeTab=activity`